app/models/sponsorship.rb

```diff
@@ -0,0 +1,19 @@
+class Sponsorship < ApplicationRecord
+
+#validate people_id
+  validates :people_id, presence: true
```

Comment:
@hyanying If we have a `people` table, the `sponsorship` table's foreign key to a person should be `person_id`, not `people_id`.  Can you change here and in the migration/schema?